Documentation/button/button.html

```diff
@@ -0,0 +1,158 @@
+      crossorigin="anonymous"
+    ></script>
+    <title>Stark Buttons</title>
+  </head>
+  <body>
+    <nav class="navigation container">
+      <div class="nav-brand">Stark UI</div>
+      <ul class="list-non-bullet list-align">
+        <li class="list-inline">
+          <a class="links link-active" href="../../index.html">Home</a>
+        </li>
+        <li class="list-inline">
+          <a class="links" href="./button.html">Docs</a>
+        </li>
+      </ul>
+    </nav>
+    <div class="flex">
+      <div class="left_div">
+        <div class="box1">
+          <p class="content">Components</p>
+          <hr />
+        </div>
+        <div class="scroll">
+          <div class="item">
+            <a class="components" href="#avatar">Avatar</a>
+          </div>
+          <div class="item">
+            <a class="components" href="#alerts">Alerts</a>
+          </div>
+          <div class="item">
+            <a class="components" href="#badge">Badge</a>
+          </div>
+          <div class="item">
+            <a class="components" href="#button">Button</a>
+          </div>
+          <div class="item">
+            <a class="components" href="#image">Image</a>
+          </div>
+          <div class="item">
+            <a class="components" href="#rating">Rating</a>
+          </div>
+          <div class="item">
+            <a class="components" href="#input">Input</a>
+          </div>
+          <div class="item">
+            <a class="components" href="#list">list</a>
+          </div>
+          <div class="item">
+            <a class="components" href="#toast">Toast</a>
+          </div>
+          <div class="item">
+            <a class="components" href="#typography">Typography</a>
+          </div>
+          <div class="item">
+            <a class="components" href="#card">Card</a>
+          </div>
+          <div class="item">
+            <a class="components" href="#navigation">Navigation</a>
+          </div>
+        </div>
+      </div>
+      <div>
+        <div>
+          <h1>Buttons<h1>
+          <h2>Solid buttons</h2>
+          <button class="btn btn-primary">Primary Button</button>
+          <button class="btn btn-secondary">Secondary Button</button>
+          <div>
+            <iframe
+              src="https://carbon.now.sh/embed?bg=rgba%28171%2C+184%2C+195%2C+1%29&t=seti&wt=none&l=javascript&width=680&ds=true&dsyoff=20px&dsblur=68px&wc=true&wa=true&pv=56px&ph=56px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=1x&wm=false&code=%253Ch2%253ESolid%2520buttons%253C%252Fh2%253E%250A%253Cbutton%2520class%253D%2522btn%2520btn-primary%2522%253EPrimary%2520Button%253C%252Fbutton%253E%250A%253Cbutton%2520class%253D%2522btn%2520btn-secondary%2522%253ESecondary%2520Button%253C%252Fbutton%253E"
+              style="
+                width: 631px;
+                height: 225px;
+                border: 0;
+                transform: scale(1);
+                overflow: hidden;
```

Comment:
You can fix the width for all the iframes you've used, this will make your library look consistent.